<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor_admin_basedialog</class>
 <widget class="QWidget" name="monitor_admin_basedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_5"/>
     </item>
     <item row="1" column="1">
      <widget class="QTreeWidget" name="mUi_qlwSelectionList">
       <property name="minimumSize">
        <size>
         <width>184</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>184</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoExpandDelay">
        <number>-1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Raw history</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FTP logs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Insertion logs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EMAIL history</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yield-Man Admin DB Events logs</string>
        </property>
        <property name="flags">
         <set>ItemIsEnabled</set>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="pageHistory">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="hBoxLayoutHistory">
           <item>
            <widget class="QLabel" name="labelFile_History">
             <property name="text">
              <string>History file:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboFiles_History">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelSearch_History">
             <property name="text">
              <string>Find:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editSearch_History"/>
           </item>
           <item>
            <widget class="QPushButton" name="buttonPrev_History">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find previous</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_left.png</normaloff>:/gex/icons/arrow_left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonNext_History">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find next</string>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_right.png</normaloff>:/gex/icons/arrow_right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTextEdit" name="textEditHistory">
           <property name="font">
            <font>
             <family>Bitstream Vera Sans Mono</family>
            </font>
           </property>
           <property name="autoFormatting">
            <set>QTextEdit::AutoAll</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageGexFtpLog">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="hBoxLayoutGexFtp">
           <item>
            <widget class="QLabel" name="labelFile_GexFtp">
             <property name="text">
              <string>Log file:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboFiles_GexFtp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonRefresh_GexFtp">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32767</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelSearch_GexFtp">
             <property name="text">
              <string>Find:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editSearch_GexFtp"/>
           </item>
           <item>
            <widget class="QPushButton" name="buttonPrev_GexFtp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find previous</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_left.png</normaloff>:/gex/icons/arrow_left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonNext_GexFtp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find next</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_right.png</normaloff>:/gex/icons/arrow_right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTreeWidget" name="mUi_qtwGexFtp">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>110</number>
           </attribute>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ftp server</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Message</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageReport">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="labelFile_Report">
             <property name="text">
              <string>Report file:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboFiles_Report">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="labelSearch_Report">
             <property name="text">
              <string>Find:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="editSearch_Report"/>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="buttonPrev_Report">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find previous</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_left.png</normaloff>:/gex/icons/arrow_left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="buttonNext_Report">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find next</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_right.png</normaloff>:/gex/icons/arrow_right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="6">
            <widget class="QTreeWidget" name="listView_Report">
             <column>
              <property name="text">
               <string>Date</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Time</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>FileName</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Comment</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>dataPump</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Directory</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Size (Mb)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Duration (s)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Speed (Mb/s)</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageGexEmailLog">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="hBoxLayoutGexEmail">
           <item>
            <widget class="QLabel" name="labelFile_GexEmail">
             <property name="text">
              <string>Log file:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboFiles_GexEmail">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonRefresh_GexEmail">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32767</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelSearch_GexEmail">
             <property name="text">
              <string>Find:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editSearch_GexEmail"/>
           </item>
           <item>
            <widget class="QPushButton" name="buttonPrev_GexEmail">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find previous</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_left.png</normaloff>:/gex/icons/arrow_left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonNext_GexEmail">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Find next</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/arrow_right.png</normaloff>:/gex/icons/arrow_right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTreeWidget" name="listView_GexEmail">
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mail file</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Message</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_poEvenLogPage">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="m_poReloadViewManager">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reload filter settings and refresh the GUI&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Reload</string>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/refresh.png</normaloff>:/gex/icons/refresh.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_poSaveViewManager">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export filter settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
             <property name="icon">
              <iconset resource="../gex.qrc">
               <normaloff>:/gex/icons/file_save.png</normaloff>:/gex/icons/file_save.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../gex.qrc"/>
 </resources>
 <connections/>
</ui>
